docs/run/advanced-runtime-options.mdx

```diff
@@ -123,21 +127,6 @@ options.optimization_level = 1
 estimator = Estimator(session=backend, options=options)
 ```
 
-Turning off all optional runtime compilation steps requires a "second-level option", as follows:
-
-```python
-from qiskit_ibm_runtime import Estimator, Options
-
-options = Options()
-options.transpilation.skip_transpilation = True
-
-estimator = Estimator(session=backend, options=options)
-```
-
-For more information and a complete list of advanced transpilation options, see the Advanced transpilation options table in the
-[Runtime compilation topic](configure-runtime-compilation#transpilation-table).
-
-
```

Comment:
The `skip_transpilation` option is still valid. I am going to restore this section.